README.md

```diff
@@ -179,6 +179,20 @@ Here is listed the basic API of both KtdGridComponent and KtdGridItemComponent.
 
 /** Minimum amount of pixels that the user should move before it starts the drag sequence. */
 @Input() dragStartThreshold: number = 0;
+
+/**
+ * Whether to use manual drag event handling for this item instead of the default automatic drag event handling. When enabled, pointer
+ * events passed to the grid item's onManualDragStart method will initiate dragging, and any KtdGridDragHandle directives will be ignored.
+ */
+@Input() enableManualDragEvents: boolean = false;
```

Comment:
I would like to avoid using this Input, check my comment in the issue: https://github.com/katoid/angular-grid-layout/issues/86#issuecomment-1563031368

Reply:
removed